diff --git idba/src/sequence/short_sequence.h idba/src/sequence/short_sequence.h
index 59bf08c..819b1c0 100644
--- idba/src/sequence/short_sequence.h
+++ idba/src/sequence/short_sequence.h
@@ -99,7 +99,7 @@ public:
         return size() < seq.size();
     }
 
-    static const uint32_t kMaxShortSequence = 128;
+    static const uint32_t kMaxShortSequence = 256;
     static const uint32_t kNumBytes = (kMaxShortSequence + 3) / 4 + 2;
 
 private:
